pages/blueprint/index.js

```diff
@@ -310,6 +318,55 @@ class BlueprintPage extends React.Component {
     const pathSuffix = cockpit.location.path[cockpit.location.path.length - 1];
     const activeKey = ["customizations", "packages", "images"].includes(pathSuffix) ? pathSuffix : undefined;
 
+    const rows = users.map(user => ({ props: { 'data-tr': user.name }, cells: [
+          user.description,
+          user.name,
+          { title: user.groups !== undefined && user.groups.includes("wheel") && <span className="fa fa-check" /> },
+          { title: user.password && <span className="fa fa-check" /> },
+          { title: (
+            user.key !== undefined && (
+              <span>
+                <span className="fa fa-check" />
+                {` `}
+                {user.key.split(" ")[2] || ""}
+              </span>
+            )
+          ) },
+        { title: (<>
```

Comment:
Can you change this `<>` to `<div>`? In the mobile layout, each child is placed in a grid column, spanning the full width, which results in the buttons being stacked. Having a parent `<div>` as the child instead of the separate buttons yields a better layout for this view.